.github/workflows/make_release.yaml

```diff
@@ -0,0 +1,44 @@
+name: create a release with a bumped version number
+
+on:
+  workflow_dispatch:
+    inputs:
+      NumberIndex:
+        description: 'Index of version number that should be bumped (for version x.y.z we have 0:x,1:y,2:z, so the default 2 is a tiny release).'
+        required: true
+        default: '2'
+      releaseBody:
+        description: 'If you want the release to have a description'
+        required: false
+
+jobs:
+  installSetup:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bump version
+      run: |
+        i=${{github.event.inputs.versionNumber}}
+        [[ "$i" == '0' ]] || [[ "$i" == '1' ]] || [[ "$i" == '2' ]] || (echo "index must be 0, 1, or 2"; exit 1)
```

Comment:
Is `set -e` defined? Otherwise `exit 1` will not cancel the action…

Reply:
I think I have seen this in other jobs that stop when the shell produces an error. Not sure how to check other than test it

Reply:
Hmyes, that sounds like `set -e` is set.